.github/workflows/check-pr.yaml

```diff
@@ -59,6 +59,9 @@ jobs:
     runs-on: tico-linux
     container:
       image: ubuntu:22.04
+      credentials:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
```

Comment:
I've checked that this grammar is working correctly on GHES.

@mhs4670go Would you please check that registered secret name is same as this code?

Reply:
Looks like `secrets` can't be tested on forked repo.